### PR TITLE
Create news page

### LINKED ISSUE #15: News Page
As a user
i want a news page
to have a overview about the newest informations in fortnite

- [x] Insert header
- [x] Insert tags
- [x] Insert news item
- [x] Insert navigation
- [x] Pull the news from the API
- [x] Refactor Component for new expectations
- [x] Create to top icon
- [x] Insert to top icon and implement functionality
- [x] Refactor Header for Fortnite Icon

### PR DESCRIPTION
**Pull request by ChristianSeelemann**
I created the news page. Unfortunately i have to made some changes cause of lacking features in the API.

Deployment: https://fortknights-page-news-uxqthbz9.herokuapp.com/
I think it looks good on every mobile phone size in the inspector.

- Insert header
- Insert tags
- Insert news item
- Insert navigation
- Pull the news from the API
- Refactor Component for new expectations
- [Create to top icon
- Insert to top icon and implement functionality
- Refactor Header for fortnite Icon

### CODE REVIEW

File: src/app/components/Card/Card.module.css [hunk 1]
```diff
@@ -1,18 +1,21 @@
 .card {
-  display: flex;
+  display: grid;
+  grid-template-columns: 1fr;
```
Comment (marwinburesch) [inline src/app/components/Card/Card.module.css:3]: Is this necessary?

File: src/app/pages/News/News.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+      }
+    ];
+  };
+}
+
+export default function News(): JSX.Element {
+  const { data: battleRoyaleData } = useFetch<NewsMotd>(
+    'https://fortnite-api.com/v2/news/br'
+  );
+  const battleRoyaleNews = battleRoyaleData?.data.motds;
```
Comment (marwinburesch) [inline src/app/pages/News/News.tsx:38]: I think this works as long as your API is working, please consider using the loading state from `useFetch`

File: src/app/components/Icons/ToTop.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+type toTopProps = {
+  color: string;
+};
+
+export default function ToTop({ color }: toTopProps): JSX.Element {
+  const scrollTop = () => {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+  };
```
Comment (marwinburesch) [inline src/app/components/Icons/ToTop.tsx:10]: does this work?

**Review by marwinburesch — COMMENTED** [review]
👍

File: src/app/components/Card/Card.module.css (hunk 1 above)
Comment (ChristianSeelemann) [reply src/app/components/Card/Card.module.css:3]: Yes. As a flexbox fanboy... but yes it is. With flexbox there was a problem with the image that was too big.

File: src/app/components/Icons/ToTop.tsx (hunk 3 above)
Comment (ChristianSeelemann) [reply src/app/components/Icons/ToTop.tsx:10]: Yes. Perfectly smoooooth 😸

**Pull request merged**